src/components/atoms/CreatableMultiSelect/CustomComponents/LabelsOption.tsx

```diff
@@ -15,16 +15,9 @@ const LabelsOption = (props: OptionProps<Option>) => {
   const {children, innerRef, innerProps, options, value} = props;
 
   const isChildren = typeof children === 'string';
-  const isCreateOption = options[0].label === children && children !== value;
+  const isCreateOption = isChildren && options[0].label === children;
 
-  const allowClick =
-    isChildren && isCreateOption
-      ? Boolean(
-          children.match(
-            /(Create (([A-Za-z0-9][-A-Za-z0-9_./]*)?[A-Za-z0-9]:\s?(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?))/g
-          )
-        )
-      : true;
+  const allowClick = isCreateOption ? children.includes(value) : true;
```

Comment:
How does it work, or how it was working? This behavior looks *very* strange.

Besides, `allowClick` may be simplified:
```ts
  const allowClick = !isCreateOption || children.includes(value);
```

Reply:
It is basically allowing to click on Create option only is value passes validation. It is achieved by formatting `formatCreateLabel` prop in `LabelsSelect` component. And here I check if this option includes the value, because in `formatCreateLabel` in add it when validation passes

Reply:
So we add some *magic* options like `Create Something` to the list of options, and then this option has some action? It sounds like this mechanism is unnecessarily obscure, instead of *actually* passing options for creation...

I'm not sure if it's worth adjusting it here with another hack - maybe it's better to keep it as it is, and just refactor soon.

Reply:
Based on https://react-select.com/props#creatable-props it should be simple:
* it may be formatted with `formatCreateLabel` prop
* it may be detected with `onCreateOption` callback